packages/playwright/test/IntegrationTests.cs

```diff
@@ -65,26 +73,44 @@ public async Task RunAxe_NoOptions()
             AxeResults axeResults = await Page!.RunAxe();
             AxeResult violation = axeResults.Violations.First();
 
-            Assert.AreEqual(violation.Id, expectedViolationId);
-            Assert.AreEqual(violation.Impact!.Value, AxeImpactValue.Serious);
-            Assert.IsFalse(string.IsNullOrEmpty(violation.Description));
-            Assert.IsFalse(string.IsNullOrEmpty(violation.Help));
+            Assert.Multiple(() =>
+            {
+                Assert.That(violation.Id, Is.EqualTo(expectedViolationId));
+                Assert.That(violation.Impact!.Value, Is.EqualTo(AxeImpactValue.Serious));
+                Assert.That(violation.Description, Is.Not.Null.Or.Empty);
+                Assert.That(violation.Help, Is.Not.Null.Or.Empty);
+            });
+        }
+
+        [Test]
+        public async Task RunAxe_WithRunOnly_RunsOnlySpecifiedTags()
+        {
+            await NavigateToPage("basic.html");
+
+            AxeRunOnly runOnly = new(AxeRunOnlyType.Tag, new List<string> { "wcag2a" });
+            AxeRunOptions options = new(runOnly);
+
+            AxeResults axeResults = await Page!.RunAxe(options);
+
+            IEnumerable<AxeResult> violationsWithoutExpectedTag = axeResults.Violations.Where(violation => !violation.Tags!.Any(tag => tag == "wcag2a"));
```

Comment:
Curious why we do `!violation...` instead of `tag != "wcag2a"`, the latter of which is similar to the test case below with `result.Id != "color-contrast"`?

Reply:
This is because `result.Tags` is a list of strings, where `result.Id` is a single string.